docs/docs/users/general/creating-campaigns.md

```diff
@@ -0,0 +1,42 @@
+---
+id: creating-campaigns
+title: Creating and managing campaigns
+---
+Here is more information about campaigns. These can be either petition campaigns, event campaigns or fundraising campaigns. 
+
+## General info
+![](https://s3-eu-west-1.amazonaws.com/bureaubolster/Wings_2019-03-25_11-31-17.png)
+
+At Campaigns, you can manage 1) Events or 2) Petitions. You can 3) click on existing campaign to edit it, or 4) Create a new one. 
+
+Events are geared towards getting people to visit an event; petitions are aimed towards letting people sign a policy proposal, letter of support or some other kind of petition. 
+
+## Where is your campaign page?
+Campaigns get their own page on your website by default; by defining the 'slug', you define its url.  Every type of campaign has its own directory, though; if your slug is 'demo-fundraiser' and your website url is mywebsite.com, the url of this specific fundraiser page can be found at 'mywebsite.com/fundraisers/demo-fundraiser'. 
+
+![](https://bolster-files.s3-eu-west-1.amazonaws.com/campaignslug2.png)
+
+
+## 'General'
+
+![](https://s3-eu-west-1.amazonaws.com/bureaubolster/Wings_2019-03-25_11-36-27.png)
+
+1. The 'General' tab is what you see on the screenshot above. Here, you can edit the general copy of the petition page. 
+2. At 'Platforms', you can configure the settings for how the website shows up in search engines and on social media.
+3. 'Meta' is sometimes used for specific tweaks under the hood - you will receive separate instructions for this if these settings are to be used.
+4. Set the title.
+5. Set the slug (the last part of the url).
+6. Change the header image.
+7. Edit the introduction copy.
+8. Edit the main copy
+9. Use the toolbar to style the main copy.
+
+## Platforms
+Here, you can set a title, meta description and meta image for Google and social media. 
+
+![](https://s3-eu-west-1.amazonaws.com/bureaubolster/Wings_2019-03-25_11-52-17.png)
+
+The general 'search engines' Title / Desciptions / image will be the default settings for social media as well; if you want to have Facebook / Twitter / WhatsApp have different values, you can set these as well - but this is optional. 
+
+A couple of additional search engine / social media settings can be configured with meta values, see meta-values#social-media-settings
```

Comment:
Wings shouldn't use an escape hatch for core features. So this doesn't belong in these docs.

Reply:
I disagree. I worded it differently, but the mechanics are the same. Wings offers an option to use a different title for Twitter than for Facebook, but it will not be used in 99.5% of the cases. The specifically designed function of this CMS part is that it allows you to optionally have fine-grained control over meta data indexed & displayed by social media platforms, but only if necessary; by default, the Search engines meta data will be used. You *always* want Twitter to display a title/description/meta image, but you *almost never* want to configure it differently than for Facebook, but it is still important that you are able to configure it differently if you want to. If anything, the UI could be improved to reflect the cascading inheriting behaviour of these settings, highlighting the fact that you can set it once for search engines *and* social media (where filling out the search engines tab autofills the other fields) whilst allowing you to override the settings inherited by default. It is not an escape hatch, it is how it should work. If anything, the CMS UI and how frontend interprets it should follow the 'editorial reality'.